cachalot/apps.py

```diff
@@ -13,7 +13,7 @@
 
 @register(Tags.compatibility)
 def check_django_version(app_configs, **kwargs):
-    if not (1, 11) <= django_version < (2, 2):
+    if not (1, 11) <= django_version < (2, 3):
```

Comment:
There will be no Django 2.3 (https://www.djangoproject.com/weblog/2015/jun/25/roadmap/). It'll be 3.0 :).

Libraries should work from 1.11 -> 2.2 with no changes needed. A 3.0 release will require we drop support for 1.11.